Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -25,7 +25,15 @@
 #include "../../../Configurations/Version.xcconfig"
 
 GCC_PREFIX_HEADER = WebCorePrefix.h;
-GCC_PREPROCESSOR_DEFINITIONS = $(DEBUG_DEFINES) BUILDING_WEBKIT GL_SILENCE_DEPRECATION=1 GLES_SILENCE_DEPRECATION=1 $(ENABLE_PDFJS) $(ENABLE_LLVM_PROFILE_GENERATION) $(inherited);
+
+WK_USE_JPEGXL = $(WK_USE_JPEGXL_$(WK_PLATFORM_NAME));
```

Comment:
I would name this `WK_USE_APPLEJPEGXL`, because this is about using `AppleJPEGXL.framework`, not about JPEG XL in general.

Reply:
I think calling it APPLEJPEGXL would also make it easier to notice when we can clean up this logic in a few years.